awscli/alias.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import logging
+import os
+import shlex
+import subprocess
+
+from botocore.configloader import raw_config_parse
+
+from awscli.utils import emit_top_level_args_parsed_event
+
+
+LOG = logging.getLogger(__name__)
+
+
+class InvalidAliasException(Exception):
+    pass
+
+
+class AliasLoader(object):
+    ALIAS_FILENAME = os.path.expanduser(
```

Comment:
That's fine with me if we expose an alias loader attribute on the clidriver. I just needed a way to hook into replace this default value.